### ISSUE TITLE
!time should either user 24h format by default or be configurable to do so

### ISSUE THREAD

**Issue by Pierakor** [PennyDreadfulMTG/Penny-Dreadful-Tools#5913]

**Comment by bakert** [comment]
Americans (our major audience) don't really do the 24-hour clock. Everyone else is used to kowtow-ing to them so I think the path of least resistance is to continue that behavior. This isn't the place where we're going to fight the hegemony of the USA (and be careful what you wish for!)

**Comment by Pierakor** [comment]
My community is 36% US based. I understand your argument though. I guess I was hoping you would care more about the convenience, configurability and usability of the bot rather than 'the path of least resistance'

**Comment by bakert** [comment]
I don't think of the bot as a thing used by any community other than the Penny Dreadful Discord. I actually assumed you were asking from the perspective of a user from that Discord. I guess I need to update my mental model of how this code is used! This is a pretty different question from the one I thought you were asking.

In general I'm loathe to include configuration options in code. Multiple paths inevitably means weaker testing and more bugs. The time command is actually a horrendous and frequently-broken piece of code despite its apparent simplicity. That said once you have a datetime with timezone information the actual formatting step is dead simple. We could add an item in config.json that lets you specify how you'd like the time output. Definitely not a priority for me but I wouldn't veto it if it got added. We do accept Pull Requests! :D

**Comment by silasary** [comment]
> I don't think of the bot as a thing used by any community other than the Penny Dreadful Discord. 

Oh, it definitely is.

Based on !notpenny stats, it's in more non-PD servers than it is PD ones.

And I see it in a lot of places where I wasn't expecting it (usually because people want the bug lookup, based on informal polling).

More per-server configuration is definitely something that's been on my backlog for a long time.  It's just a PITA to deal with.

**Comment by silasary** [comment]
Ok, this is in the code.  Unfortunately the entire build chain is blocked by #5906, so it's not live yet.
